src/components/Promise/Radar.js

```diff
@@ -28,7 +29,11 @@ const useStyles = makeStyles(({ typography, breakpoints }) => ({
   },
 }));
 
-function Radar(props) {
+function Radar({ location: locationProps, ...props }) {
+  const location = locationProps || [
+    process.en.NEXT_PUBLIC_PROMISE_LAT,
+    process.en.NEXT_PUBLIC_PROMISE_LONG,
+  ];
```

Comment:
Move this into `getStaticProps` or even in `lib` functions ... components should depends just on props if possible.